clowdapp.yaml

```diff
@@ -95,7 +95,7 @@ objects:
         - name: SECRET_KEY_BASE
           value: "secret_key_base"
     deployments:
-    - name: backend
+    - name: backend-clowder
```

Comment:
Only this backend deployment conflicts because it is not named `compliance-backend`. The rest have `compliance-` appended and do not conflict. i.e. we have `compliance-sidekiq` in production but `compliance-compliance-sidekiq` with clowder.

Reply:
@akofink  I think `prometheus-exporter`also have this problem ? I'll be removing the prefixes once we're only using clowder anyways . 

for the sake of consistency, I'll add "compliance" to both backend and prometheus exporter so clowder pods have all the same "name scheme" if you agree

Reply:
I thought we're adding suffix `-clowder` to everywhere.